packages/fluent-theme/src/components/preChatActivity/StarterPromptsCardAction.tsx

```diff
@@ -26,39 +27,41 @@ const StarterPromptsCardAction = ({ className, messageBackAction }: Props) => {
     [messageBackAction?.text, renderMarkdownAsHTML]
   );
 
+  const disabled = uiState === 'disabled';
+
   const handleClick = useCallback(() => {
     setSendBoxValue(inputTextRef.current);
 
     // Focus on the send box after the value is "pasted."
     focus('sendBox');
   }, [focus, inputTextRef, setSendBoxValue]);
 
-  return (
+  return uiState === 'blueprint' || !messageBackAction ? (
+    <div
+      className={cx(className, classNames['pre-chat-message-activity__card-action-box'])}
+      data-testid={testIds.preChatMessageActivityStarterPromptsCardAction}
+    />
+  ) : (
     <button
       className={cx(className, classNames['pre-chat-message-activity__card-action-box'])}
       data-testid={testIds.preChatMessageActivityStarterPromptsCardAction}
-      disabled={!messageBackAction}
-      onClick={handleClick}
       type="button"
+      {...(disabled ? { 'aria-disabled': 'true', tabIndex: -1 } : { onClick: handleClick })}
     >
-      {messageBackAction && (
-        <React.Fragment>
-          <div className={classNames['pre-chat-message-activity__card-action-title']}>
-            {'title' in messageBackAction && messageBackAction.title}
-          </div>
-          {'image' in messageBackAction && messageBackAction.image && (
-            <MonochromeImageMasker
-              className={classNames['pre-chat-message-activity__card-action-image']}
-              src={messageBackAction.image}
-            />
-          )}
-          <div
-            className={classNames['pre-chat-message-activity__card-action-subtitle']}
-            // eslint-disable-next-line react/no-danger
-            dangerouslySetInnerHTML={subtitleHTML}
-          />
-        </React.Fragment>
+      <div className={classNames['pre-chat-message-activity__card-action-title']}>
+        {'title' in messageBackAction && messageBackAction.title}
+      </div>
+      {'image' in messageBackAction && messageBackAction.image && (
```

Comment:
Should we make all optional? If none is passed `:empty` selector is triggered.

Reply:
Not relevant, please ignore this :smile:

Reply:
I think `title` is mandatory. `image` and `subtitle` aren't.

In the spec, either `image` or `title` is mandatory. But having `image` without `title` will break accessibility so I would leave `title` mandatory.